recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+- conda-forge/label/exceptiongroup_dev,conda-forge
```

Comment:
You should be able to take this file back out.

Reply:
Done. Thank you!